### ISSUE TITLE
Task Module doesn't show iframe in O365 connector card 

### ISSUE THREAD

**Issue by yashpariharcrest** [MicrosoftDocs/msteams-docs#1997]
I want to show a task module when clicked on a action button. i have tried deep link approach. Below is the connectorcard Json

{
    "@type": "MessageCard",
    "@context": "http://schema.org/extensions",
    "summary": "Card \"Test card\"",
    "themeColor": "0078D7",
    "title": "Card created: \"Name of card\"",
    "sections": [
        {
            "activityTitle": "David Claux",
            "activitySubtitle": "9/13/2016, 3:34pm",
            "activityImage": "https://connectorsdemo.azurewebsites.net/images/MSC12_Oscar_002.jpg",
            "facts": [ ],
            "text": "Lorem ipsum "
        }
    ],
    "potentialAction": [
        {
            "@type": "OpenUri",
            "name": "View it",
            "targets": [
                { "os": "default", "uri": "https://teams.microsoft.com/l/task/ec6a13b7-1527-4f31-b2a9-8ebee9520d15? 
 url=http://localhost:3978/static/views/connectorConfig.html&height=800&width=800&title=Suggest" }
            ]
        }
    ]
}

In Potential action, i have added an action "openURI" with a deep link. 
When clicked on the button, a task module do open BUT the iframe (url given html) doesnt show rather it shows blank. 

![taskmoduleempty](https://user-images.githubusercontent.com/62649552/90633831-65b16700-e244-11ea-9df2-44141f27684e.PNG)

So basically i want to show a task module with html rendering on click of connector card button.

**Comment by VaraPrasad-MSFT** [comment]
@yashpariharcrest, You need to use the tunneling service and pass that url tunnel url rather than passing the below local host url.
url=http://localhost:3978/static/views/connectorConfig.html&height=800&width=800&title=Suggest
This should solve your problem. Let me know if you are still facing this issue.

**Comment by yashpariharcrest** [comment]
@VaraPrasad-MSFT , now we have actually changed from connector card to adaptive card.  :)  . Anyways thanks. 👍
